### PR TITLE
2.13.5 release notes

### PR DESCRIPTION
**Pull request by SethTisue**
[rendered view](https://github.com/scala/scala-dev/blob/notes-2.13.5/releases/2.13.5.md)

this is a draft, feedback is highly welcome!

if all goes well, we will publish the release on Monday

(but feedback is still welcome even after release, edits will remain possible)

### CODE REVIEW

File: releases/2.13.5.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+As usual for our minor releases, Scala 2.13.5 is binary-compatible with the whole Scala 2.13 series.
+
+Upgrading from 2.12? Enable `-Xmigration` while upgrading to request migration advice from the compiler.
+
+## Contributors
+
+A big thank you to everyone who's helped improve Scala by reporting bugs, improving our documentation, spreading kindness in discussions around Scala, and submitting and reviewing pull requests! You are all magnificent.
+
+This release was brought to you by 23 contributors, according to `git shortlog -sn --no-merges HEAD ^v2.13.4 ^2.12.x`. Thank you Seth Tisue, A. P. Marki, Dale Wijnand, NthPortal, Jamie Thompson, Lukas Rytz, Martijn Hoekstra, Georgi Krastev, Jason Zaugg, Jasper Moeys, Sakib Hadziavdic, Anatolii Kmetiuk, Arnaud Gourlay, Marcono1234, Chia-Ping Tsai, Mike Skells, Stefan Zeiger, Waleed Khan, Yann Bolliger, Guillaume Martres, 梦境迷离, Ethan Atkins, Da.
```
Comment (propensive) [inline releases/2.13.5.md]: Is `Da` a person?

Comment (SethTisue) [reply releases/2.13.5.md]: it's Darcy Shen, @da-tubi. I'll fix it and I'll also add a mailmap entry so it happens automatically next time.

Comment (SethTisue) [reply releases/2.13.5.md]: https://github.com/scala/scala/pull/9511

**Comment by martijnhoekstra** [comment]
I cleaned up the PR text of https://github.com/scala/scala/pull/9343 a bit further.

**Review by dwijnand — APPROVED** [review]

**Comment by SethTisue** [comment]
These notes are now in place at https://github.com/scala/scala/releases/tag/v2.13.5 — but they can still be improved. I'll continue responding to any feedback here.

**Pull request closed without merging**